crates/sncast/tests/e2e/invoke.rs

```diff
@@ -52,6 +52,45 @@ async fn test_happy_case(account: &str) {
     assert!(matches!(receipt, Invoke(_)));
 }
 
+#[test_case("oz_cairo_0"; "cairo_0_account")]
+#[test_case("oz_cairo_1"; "cairo_1_account")]
+#[test_case("oz"; "oz_account")]
+#[test_case("argent"; "argent_account")]
+#[test_case("braavos"; "braavos_account")]
+#[tokio::test]
+
+async fn test_happy_case_common_arguments_after_subcommand(account: &str) {
+    let args = vec![
+        "invoke",
+        "--accounts-file",
+        ACCOUNT_FILE_PATH,
+        "--account",
+        account,
+        "--int-format",
+        "--json",
+        "--url",
+        URL,
+        "--contract-address",
+        MAP_CONTRACT_ADDRESS_SEPOLIA,
+        "--function",
+        "put",
+        "--calldata",
+        "0x1 0x2",
+        "--max-fee",
+        "99999999999999999",
+        "--fee-token",
+        "eth",
+    ];
+
+    let snapbox = runner(&args);
```

Comment:
```suggestion
    runner(&args).assert().success();
```

It is enough to check if it works. Still, we kind of check the clap functionality, so I don't think this test is necessary at all.

Reply:
Suggestion is reasonable, we can just check if it succeeds. 
This test will fail if we stop supporting the `global` (for some reason), so I think we should keep it.